src/ucp/wireup/wireup_cm.c

```diff
@@ -305,6 +323,13 @@ static ssize_t ucp_cm_client_priv_pack_cb(void *arg,
 
     /* At this point the ep has only CM lane */
     ucs_assert((ucp_ep_num_lanes(ep) == 1) && ucp_ep_has_cm_lane(ep));
+
+    UCP_CM_CB_CHECK_UCP_EP(ep, ucp_ep_get_cm_uct_ep(ep),
+                           {
+                               status = UCS_ERR_NOT_CONNECTED;
```

Comment:
seems like the error code does not reflect actual one, we had status in EP config key for such cases and probably deleted this...

Reply:
yes, we deleted it, but we set nothing to the `status`, it was UCS_OK always.

Reply:
also, I think `status` should be stored in UCP control extension rather in EP config key
`status` belongs to EP, but not to all EPs which use this EP config key

Reply:
the ep assumed to be reconfigured on failure with relevant status, we don't expect a lot of reasons of failure and all failed EPs with the same status should have the same configuration

Reply:
- assert that if FAILED, also CLOSED
- return ERR_CANCELED